codegen/templates/chart/operator-deployment.yamltmpl

```diff
@@ -102,17 +102,15 @@ spec:
             memory: 256Mi
 {{- end}}
         securityContext:
-[[- if $container.SecurityContext ]]
-[[ toYaml $container.SecurityContext | indent 10 ]]
-[[- else ]]
           runAsNonRoot: true
-          runAsUser: 10101
+{{- if not $[[ $operatorVar ]].floatingUserId }}
```

Comment:
correct the indentation, same for `codegen/test/chart/templates/deployment.yaml`